chef/automate
# Removing duplicate requests

### :nut_and_bolt: Description: What code changed, and why?

Removing duplicate API requests from the policy and team pages. When loading the pages the observable subscribe functions were being called multiple times for with the same data. This caused the page's data to be requested each time.

### :chains: Related Resources

#2104

### :+1: Definition of Done
All of the auth pages do not perform multiple requests for page data. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-ui && start_all_services`
1. Open the Teams detail page for the admin https://a2-dev.test/settings/teams/admins
1. Open the Chrome inspect tool then select the network tab. 
1. Ensure each API request is only called once.
1. Do the same thing for the team add user page, policy details page, and policy add member page. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
